spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-core.spec.js

```diff
@@ -629,36 +629,34 @@ describe('GA4 core', function () {
         expect(builtEcommerceObject).toEqual(expectedEcommerceObject)
       })
 
-      it('the ecommerce items array is limited to 200 items', function () {
+      it('the ecommerce items array is limited to 15,000 bytes', function () {
         var innerHTML = ''
         var ecommerceItems = []
 
-        // Adds 500 search results to the DOM, but only adds 200 to our expected ecommerce object array
-        for (var i = 0; i < 500; i++) {
+        for (var i = 0; i < 4000; i++) {
           innerHTML = innerHTML + '<a data-ga4-ecommerce-path="https://www.gov.uk/the-warm-home-discount-scheme" href="https://www.gov.uk/the-warm-home-discount-scheme" data-ga4-ecommerce-index="' + (i + 1) + '">Check if you’re eligible for the Warm Home Discount scheme</a>'
 
-          if (i < 200) {
-            ecommerceItems.push({
-              item_id: 'https://www.gov.uk/the-warm-home-discount-scheme',
-              item_content_id: undefined,
-              item_list_name: 'Smart answer results',
-              index: i + 1
-            })
-          }
+          ecommerceItems.push({
+            a: 'b'
+          })
         }
 
-        resultsCount.innerHTML = '500 results'
+        resultsCount.innerHTML = '4000 results'
         results.innerHTML = innerHTML
```

Comment:
Is `results` used anywhere? Wondering if line 637's HTML chunk is needed now?

Reply:
@andysellick I've deleted line 644, but I can't remove the HTML chunk as the eCommerce tracking needs the HTML to exist, as that's what it uses to populate its GA4 data

Reply:
Ah okay no worries. Just couldn't see where it was used and wanted to check.